content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.Network
+**Derived From:** [cloudify.nodes.Network]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP Network. This supports either auto-assigned or manual subnets. Legacy networks are not supported. See the GCP Manager and Networks section below if you plan to run a cloudify manager on GCP.
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+  * `auto_subnets`
+    Whether to use the GCP "autoCreateSubnetworks" feature (see https://cloud.google.com/compute/docs/subnetworks#networks_and_subnetworks)
+
+    *default:* True
+  * `additional_settings`
+    Additional setting for network
+
+    *default:* {}
+  * `name`
+    Optional Network name. The instance ID will be used by default.
+
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+
+
+
+### Example
+
+{{< gsHighlight  yaml  >}}
+
+my_net:
+  type: cloudify.gcp.nodes.Network
+
+{{< /gsHighlight >}}
+
+
+
+
+
+## cloudify.gcp.nodes.Route
+**Derived From:** [cloudify.nodes.Router]({{< relref "blueprints/built-in-types.md" >}})
+
+A defined route, which will be added to the specified network.
+If tags are specified, it will only be added to Instances matching them.
+
+
+
+**Properties:**
+
+
+  * `dest_range`
+    The outgoing range that this route will handle
+
+    *required* None
+  * `priority`
+    The routing table priority for this route. Routes with lower priority numbers will be chosen first if more than one route with a matching prefix of the same length.
+
+    *default:* 1000
+  * `additional_settings`
+    Additional setting for firewall
+
+    *default:* {}
+  * `next_hop`
+    The Instance, IP or VpnTunnel which will handle the matching packets
+
+    *default:*
+  * `name`
+    Optional Route name. The instance ID will be used by default.
+
+    *default:*
+  * `tags`
+    Instance tags that this route will be applied to
+
+    *default:* []
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+
+
+
+
+
+## cloudify.gcp.nodes.SecurityGroup
+**Derived From:** [cloudify.nodes.SecurityGroup]({{< relref "blueprints/built-in-types.md" >}})
+
+A virtual SecurityGroup.
+
+Google Cloud Platform has no entity equivalent to a Security Group on AWS or OpenStack, so as a convenience Cloudify includes a virtual one. It is implemented behind the scenes using a specially constructed tag and a number of FirewallRules.
+
+
+
+**Properties:**
+
+
+  * `rules`
+    List of FirewallRules which will form this SecurityGroup. Only the `sources:` and `allowed:` fields should be supplied (see FirewallRule properties for details).
```

Comment:
List of FirewallRules ~~which~~ **that** will form this ~~SecurityGroup~~ **security group**.